src/braintree.webhook.controller.ts

```diff
@@ -29,7 +29,6 @@ export default class BraintreeWebhookController {
 
   async handle(@Req() request) {
```

Comment:
this needs to be formatted, i think !

Reply:
no I've ran format on it. What's the problem? The decorator?

Reply:
ah think I see now!

Reply:
ah I think see now!